components/auth/login-form.tsx

```
@@ -39,84 +44,86 @@ export default function LoginForm() {
-            Email Address
-          </Label>
-          <Input
-            id="email"
-            name="email"
-            type="email"
-            required
-            className="input-neon mt-1"
-            placeholder="your.email@awesome.com"
-          />
-        </motion.div>
+          {showEmailLogin ? 'Hide admin login' : 'Admin or registered user?'}
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)